bokehjs/test/common/defaults.coffee

```diff
@@ -178,12 +186,12 @@ describe "Defaults", ->
       python_defaults = get_defaults(name)
       coffee_defaults = attrs
       if not check_matching_defaults(name, python_defaults, coffee_defaults)
-        console.log(name)
-        #console.log('python defaults:')
-        #console.log(python_defaults)
-        #console.log('coffee defaults:')
-        #console.log(coffee_defaults)
-        console.log(_.difference(_.keys(python_defaults), _.keys(coffee_defaults)))
+        # console.log(name)
+        # console.log('python defaults:')
+        # console.log(python_defaults)
+        # console.log('coffee defaults:')
+        # console.log(coffee_defaults)
+        # console.log(_.difference(_.keys(python_defaults), _.keys(coffee_defaults)))
```

Comment:
You commented this out, but this reporting was extremely helpful to me, and I couldn't find another way of getting correct reporting.